code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -111,7 +110,7 @@
 	to_chat(src, "<b>You are a swarmer, a weapon of a long dead civilization. Until further orders from your original masters are received, you must continue to consume and replicate.</b>")
 	to_chat(src, "<b>Clicking on any object will try to consume it, either deconstructing it into its components, destroying it, or integrating any materials it has into you if successful.</b>")
 	to_chat(src, "<b>Ctrl-Clicking on a mob will attempt to remove it from the area and place it in a safe environment for storage.</b>")
-	to_chat(src, "<b>Prime Directives:</b>")
+	to_chat(src, "<b>Prime Directives:")
```

Comment:
Any reason you removed the bold closing tag here

Reply:
Collateral damage from removing the tags from the description of the spawner. My bad.